src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,18 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
 
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+        book.setTitle("BookTitle");
+        book.setPrice(new BigDecimal(152));
+        bookDao.create(book);
```

Comment:
According to the task description, we need to show all the CRUD operations in the main.